### PR TITLE
WIP: User guide for running redis container

### PR DESCRIPTION
**Pull request by aravindavk**
using Gluster Volume as persistent store

TODO:
- Yet to be validated with CSI driver setup, tried locally with
  custom setup.
- README from #16 need to be converted as setup guide once merged

Signed-off-by: Aravinda VK <avishwan@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@aravindavk I dont know whats special about redis, ideally we should have an example app which can be anything.  So, we dont have to name it saying `using-redis` guide..

**Comment by aravindavk** [comment]
> @aravindavk I dont know whats special about redis, ideally we should have an example app which can be anything. So, we dont have to name it saying using-redis guide..

Nothing special about the redis, redis example was present in the README of #16, I took it to completion.

We need similar document for web server, postgresql and others.

**Comment by aravindavk** [comment]
> @aravindavk I dont know whats special about redis, ideally we should have an example app which can be anything. So, we dont have to name it saying using-redis guide..

Surprised to see your comment about the redis. Even your PR(https://github.com/gluster/gluster-csi-driver/pull/11/files#diff-04c6e90faac2675aa89e2176d2eec7d8R121) contains the redis example

File: docs/using-redis.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Gluster Volume as persistent store for running Redis server
```
Comment (JohnStrunk) [inline docs/using-redis.md]: New suggested title: "Using a Gluster volume for persistent application storage"

File: docs/using-redis.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+the claim size as required
+
+```
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: redis-server-vol
+  annotations:
+    volume.beta.kubernetes.io/storage-class: glusterfs-csi
```
Comment (JohnStrunk) [inline docs/using-redis.md]: Please change this is the modern syntax for specifying the StorageClass. See: https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/#using-dynamic-provisioning

**Comment by Madhu-1** [comment]
ok to test

**Comment by humblec** [comment]
retest this please.

**Comment by humblec** [comment]
ok to test

**Comment by nigelbabu** [comment]
retest this please

**Comment by Madhu-1** [comment]
CI is working fine :+1: @nigelbabu  Thank you

**Comment by humblec** [comment]
@aravindavk are you polishing this PR with review comments ?

**Comment by aravindavk** [comment]
> @aravindavk are you polishing this PR with review comments ?

Sure. I will refresh the PR soon

**Pull request closed without merging**